content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,106 @@
+managers [Shamsher Ansari](https://github.com/shamsher31) (Red Hat), [Steven Landlow](https://github.com/stevenctl)
+(Google) and [Jacob Delgado](https://github.com/jacob-delgado) (Aspen Mesh) for helping to get Istio 1.9.0
+published.
+
+Here are some highlights for this release:
+
+## Virtual Machine Integration (Beta)
+
+Enabling workloads running in VMs to be part of the Istio service mesh, being able to apply consistent policy, and
+collect telemetry across containers and VMs has always been a focus of the Istio community.  We have continued improving
+the stability, testing and documentation for VM integration, and are happy to announce that in Istio 1.9 we have
+promoted this feature to Beta.
+
+Here's a list of supporting documents which you can follow to easily expand your Istio service mesh to include VMs:
+
+* [Virtual Machine Installation](/docs/setup/install/virtual-machine/) to get started.
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
+* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
+* [Bookinfo with a Virtual Machine](/docs/examples/virtual-machines/) to learn more about connecting virtual machine workloads to Kubernetes workloads.
+
+## Request Classification (Beta)
+
+Istio continues to make mesh telemetry collection more configurable. In this release,
+[Request Classification](/docs/tasks/observability/metrics/classify-metrics/) has been promoted to Beta. This feature
+enables users to more precisely understand and monitor the traffic in their service mesh.
+
+## Kubernetes Service API support (Alpha)
+
+Support for [Kubernetes `service-apis`](https://kubernetes-sigs.github.io/service-apis/) has been an active area of
+development since Istio 1.8. These are marketed as the next generation of the Ingress API, and draw inspiration from
+Istio's `Gateway` and `VirtualService`. The Istio community is eager to evolve with Kubernetes and these CRDs in upcoming
+releases. To try them out, check out the [getting started documentation](/docs/tasks/traffic-management/ingress/service-apis/).
+These APIs are still evolving, so feedback is welcome!
+
+## Better external authorization (Experimental)
+
+Authorization policy now supports an experimental feature of
+[`CUSTOM` action](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action) that allows users to
+integrate with external auth system (e.g. OPA, OAuth2, etc.) more easily.
+
+We will soon be publishing a blog on better external authorization, but for now you can look at our documentation,
+[external authorization with custom action task page](/docs/tasks/security/authorization/authz-custom) to use this
+feature. If you are using envoy filter today to integrate with an external auth system, we would recommend you try this
```

Comment:
I think we mean EnvoyFilter [link]

If you are using the EnvoyFilter API today... we recommend ...

@yangminzhu

Reply:
Updated.